lib/screens/StartupScreen.dart

```diff
@@ -946,19 +1214,19 @@ class _StartPageState extends State<StartPage> {
                 if (_code.value.text.isEmpty) {
                   setState(() => unfilledCode = true);
                 } else {
+                  _code.clear();
                   setState(() {
-                    _code.clear();
-                    state = 0;
+                    Navigator.pushNamedAndRemoveUntil(
+                        context, '/food', (Route<dynamic> route) => false);
```

Comment:
Looks like this doesn't actually verify the code - can you add a TODO to fix that later or update this to actually use the code?